bundle/manifests/rhods-operator.clusterserviceversion.yaml

```diff
@@ -29,7 +29,7 @@ metadata:
                 "managementState": "Managed"
               },
               "kserve": {
-                "managementState": "Removed"
+                "managementState": "Managed"
               },
               "modelmeshserving": {
                 "managementState": "Managed"
```

Comment:
switch to "Removed" ?

Reply:
ah, i see. this is from the example! i will fix it.